website/content/en/releases/13.1R/relnotes.adoc

```diff
@@ -144,14 +160,15 @@ This section describes changes that affect networking in FreeBSD.
 [[network-general]]
 === General Network
 
+…
 
```

Comment:
[[hardware]]
=== Hardware Support

… 

[[hardware-virtualization]
==== Virtualization Support

_Bhyve improvements_ – <https://forums.freebsd.org/posts/557777>

Reply:
Bhyve: <https://www.freebsd.org/releases/13.1R/relnotes/#userland-programs> (under _Userland Application Changes_, not under _Virtualization Support_).